actors/cache/actors_cache.go

```diff
@@ -66,21 +66,23 @@ func (a *ActorsCache) ClearBadAddressCache() {
 	a.badAddress.Clear()
 }
 
-func (a *ActorsCache) GetActorCode(add address.Address, key filTypes.TipSetKey) (string, error) {
+func (a *ActorsCache) GetActorCode(add address.Address, key filTypes.TipSetKey, onChainOnly bool) (string, error) {
 	// Check if this address is flagged as bad
 	if a.isBadAddress(add) {
 		return "", fmt.Errorf("address %s is flagged as bad", add.String())
 	}
 
-	// Try kv store cache
-	actorCode, err := a.offlineCache.GetActorCode(add, key)
-	if err == nil {
-		return actorCode, nil
+	if !onChainOnly {
+		// Try kv store cache
+		actorCode, err := a.offlineCache.GetActorCode(add, key)
```

Comment:
Not critical for this issue, but the ActorsCache should be accessed through the wrapper API like this [method](https://github.com/Zondax/fil-parser/blob/7788d0141406bed7542921d306cdc04d58cc338f/actors/cache/actors_cache.go#L69), just to be backend independent.

Reply:
I just took what I was there before and add a new "feature" to this piece of code. I think the actors cache is being accessed with that method you mentioned